helm/frconfig/README.md

```diff
@@ -1,41 +1,55 @@
 # frconfig - Manage configuration for the ForgeRock platform components
 
-This chart is responsible for fetching configuration to a shared
-persistent volume that is mounted by other pods. It must be running
-*before* any other components can be deployed.
+This chart creates Kubernetes config maps and secrets needed to clone platform configurations
+from a git repository.
 
-The shared PVC *must* be a read-write-many volume type. NFS is a popular option here, but
-any read-write-many volume will work.
-
-The script ../../bin/create-nfs-provisioner provides a sample NFS server + dynamic volume provisioner that
-is suitable for this purpose.
+This is a prerequesite chart that must be deployed before other charts such as openig, amster, and openidm.
 
 ## values.yaml
 
-The defaults provided in values.yaml will clone the "public" forgeops-init git repository. This is a bare bones
-starter repository with a minimal platform configuration. 
+The defaults in values.yaml clone the public (read only) [forgeops-init](https://github.com/ForgeRock/forgeops-init) repository. This 
+is a bare bones starter repository with a minimal platform configuration. A https: git URL is used to clone the configuration, and
+it does not require any git ssh secrets to perform the pull.  
+
+To use a different git repository, you must update values.yaml with the appropriate git URLs. 
+Use a git:// url for private repostories that require ssh access, as http only works for public, read-only repositories.
 
-If you want to use a custom configuration, create a custom.yaml file that overrides the defaults. A sample
+To use a custom configuration, create a custom.yaml file that overrides the defaults. A sample
 is shown below:
 
 ```yaml
 git:
   # git repo to clone.
   repo: "git@github.com:Acme/cloud-deployment-config.git"
-  branch: master
-  # Name of a secret that contains a key "id_rsa"
-  # The secret contains the git ssh key that has permissions to clone and/or update the git repo.
-  sshSecretName: "git-ssh-key"
+  branch: mybranch
+# Usually you do not need to change config.name. See below
+# config:
+#   name: frconfig
+```
+
+## git secret
+
+A dummy ssh secret `id_rsa` is stored in the `frconfig` secret. If you need ssh access to your git repository
+you must replace this secret with a real ssh key. For example:
+
+```shell
+# Generate your own id_rsa and id_rsa.pub keypair, according to the instructions on github or stash.
```

Comment:
You could remove ", according to the instructions on github or stash"

If you want to have something, you could say "following you Git repository provider's instructions."